src/main/java/org/terasology/servermotd/ServerMOTDSystem.java

```diff
@@ -16,30 +16,55 @@
 package org.terasology.servermotd;
 
 import org.terasology.context.Context;
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
-import org.terasology.network.NetworkMode;
-import org.terasology.network.NetworkSystem;
+import org.terasology.logic.console.commandSystem.annotations.Command;
+import org.terasology.logic.console.commandSystem.annotations.CommandParam;
+import org.terasology.logic.console.commandSystem.annotations.Sender;
 import org.terasology.registry.CoreRegistry;
-import org.terasology.registry.Share;
+import org.terasology.registry.In;
+
+
+@RegisterSystem(RegisterMode.AUTHORITY)
+public class ServerMOTDSystem extends BaseComponentSystem {
+    @In
+    private EntityManager entityManager;
 
-@Share(ServerMOTD.class)
-@RegisterSystem(RegisterMode.ALWAYS)
-public class ServerMOTDSystem extends BaseComponentSystem implements ServerMOTD {
     private Context context = CoreRegistry.get(Context.class);
 
-    @Override
+    private EntityRef MOTDEntity;
```

Comment:
Bad variable name here, should probably be `motdEntity` 

Generally when using acronyms you treat them as a word for capitalization reasons, although that's a pretty soft role when the acronym does look nice - but here it doesn't really and clashes with the hard rule to always start a regular variable with a lowercase letter :-)